inspirehep/dojson/hep/fields/bd5xx.py

```diff
@@ -64,11 +64,11 @@ def public_notes2marc(self, key, value):
     }
 
 
-@hep.over('hidden_notes', '^595..')
-def hidden_notes(self, key, value):
-    """Hidden notes."""
-    def _hidden_notes(value):
-        def _hidden_note(value, a=None):
+@hep.over('_private_notes', '^595..')
+def private_notes(self, key, value):
+    """Private notes."""
+    def _private_notes(value):
+        def _private_note(value, a=None):
             source = value.get('9')
             if value.get('c'):
```

Comment:
I think this is a very good idea. Indeed we are having these usecases. Additional one is: "candidate to be exported to". (somehow this was always implemented with special collections but to make them explicit in the data model allows to build more straightforward workflows.